src/pages/academy/components/video-banner/carousel/_carousel-style.ts

```diff
@@ -86,11 +86,10 @@ export const StyledChevron = styled((props) => <Chevron {...props} />)`
 
     path {
         ${(props) => {
-            const black_color = 'fill: var(--color-black);'
-            const red_color = 'fill: var(--color-red);'
+            const choosed_color = `fill: var(--color-${props.color});`
```

Comment:
```suggestion
            const chosen_color = `fill: var(--color-${props.color});`
```
*selected_color would also do 🙂